lib/web/ui/usercontext_test.go

```diff
@@ -70,6 +70,8 @@ func (s *UserContextSuite) TestNewUserContext(c *check.C) {
 	c.Assert(userContext.ACL.Users, check.DeepEquals, denied)
 	c.Assert(userContext.ACL.Tokens, check.DeepEquals, denied)
 	c.Assert(userContext.ACL.Nodes, check.DeepEquals, denied)
+	c.Assert(userContext.ACL.Request.RequestStrategy, check.Equals, services.RequestStrategyOptional)
```

Comment:
With structural testing:
```go
c.Assert(userContext.ACL.Request, check.DeepEquals, requestAccess{
	RequestStrategy: services.RequestStrategyOptional,
})
```
which validates in a more readable manner and protects against future modifications to `Request` that omit changes to the testcase and will go unnoticed otherwise.

Reply:
revised